extensions/adapters/vector/src/main/java/mil/nga/giat/geowave/adapter/vector/plugin/GeoWaveFeatureReader.java

```diff
@@ -260,13 +263,17 @@ public BaseIssuer(
 		public CloseableIterator<SimpleFeature> query(
 				final PrimaryIndex index,
 				final mil.nga.giat.geowave.core.store.query.Query query ) {
+			final QueryOptions queryOptions = new QueryOptions(
```

Comment:
Could simplifying this by having a new constructor that takes a List<String> for the subset attributes where getSubset() can return an empty list when subseting is not requested?